tests/core/test_processor.py

```diff
@@ -78,13 +78,14 @@
     with pytest.warns(UserWarning) as record:
         default_processor._log_unseen_features(parsed)
     assert len(record) == 2
+
     assert (
-        record[0].message.args[0]
-        == "Interpreter parsed an intent 'dislike' that is not defined in the domain."
+        record[0].message.args[0].startswith("Interpreter parsed an intent 'dislike'")
```

Comment:
Thinking out loud: That can be tough in the future to have the warring text in different places: in the original business logic file and in the test.

Reply:
yes very true, I think this test shouldn't check for the actual message, checking if a warning is raised should be enough :+1: